CHANGELOG.rst

```diff
@@ -2,7 +2,13 @@
 CHANGELOG
 =========
 
-2024.2 (unreleased)
+1.0.0 (UNRELEASED)
+================
+
+* We have decided internally to revert the versioning schema back to SemVer, starting at ``1.0.0``, in order to improve better backward and forward compatibility support for plugins.
```

Comment:
Unrelated to this PR.

We probably should add some code so future alfasim versions treat plugins build using the sdk 2024.X series as some version from the range 0.21.0-0.30.0 (some thing in those lines).

https://esss.atlassian.net/browse/ASIM-5863